Dockerfile

```diff
@@ -127,20 +127,9 @@ FROM base as dev
 
 ENV INVENTREE_DEBUG=True
 
-ENV INVENTREE_DEV_DIR="${INVENTREE_HOME}/dev"
-
 # Location for python virtual environment
 # If the INVENTREE_PY_ENV variable is set, the entrypoint script will use it!
-ENV INVENTREE_PY_ENV="${INVENTREE_DEV_DIR}/env"
-
-# Override default path settings
-ENV INVENTREE_STATIC_ROOT="${INVENTREE_DEV_DIR}/static"
-ENV INVENTREE_MEDIA_ROOT="${INVENTREE_DEV_DIR}/media"
-ENV INVENTREE_PLUGIN_DIR="${INVENTREE_DEV_DIR}/plugins"
-
-ENV INVENTREE_CONFIG_FILE="${INVENTREE_DEV_DIR}/config.yaml"
```

Comment:
@SchrodingersGat are you sure we do not need any of those variables?

Reply:
These variables are already defined at the top of the dockerfile. We were (previously) re-defining them for the development build but there is no longer any need to do this.